packages/block/test/mergeBlock.spec.ts

```diff
@@ -40,6 +40,11 @@ tape('The Merge tests', function (t) {
     st.end()
   })
 
+  t.test('should return that EIP 3675 is activated', function (st) {
+    st.ok(common.isActivatedEIP(3675))
```

Comment:
I guess this test is unnecessary, this is such common functionality and tested elsewhere. Will remove.